docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -98,6 +99,69 @@ for instance the MediaBundle defines the method as::
         }
     }
 
+.. note::
+    In your own admin, media is just a field and not the ``$object``. Therefore,
+    the code above must be updated this way:
+
+    .. code-block:: php
+
+        <?php
+
+            ...
+
+            public function getObjectMetadata($object)
+            {
+                $media = $object->getMediaField();
+
+                $provider = $this->pool->getProvider($media->getProviderName());
+
+                $url = $provider->generatePublicUrl($media, $provider->getFormatName($media, 'admin'));
+
+                return new Metadata($media->getName(), $media->getDescription(), $url);
+            }
+
+    Also, the ``$pool`` variable is not available so you will have to replace
+    ``$provider = $this->pool->getProvider($media->getProviderName());`` by
+    ``$provider = $this->getConfigurationPool()->getContainer()->get($object->getProviderName());``.
+
+    Another better option is to use dependency injection. For this, first define
+    the ``$pool`` variable and override the constructor:
+
+    .. code-block:: php
+
+        <?php
+
+            ...
```

Comment:
Please use ‘// ...’ to have valid php code same below

Reply:
Done